src/Chainweb/Chainweb/MinerResources.hs

```diff
@@ -68,7 +70,8 @@ data MiningCoordination logger cas = MiningCoordination
     , _coordCutDb :: !(CutDb cas)
     , _coordState :: !(TVar MiningState)
     , _coordLimit :: !Int
-    , _coord503s :: IORef Int }
+    , _coord503s :: IORef Int
+    , _coordChains :: TVar [ChainId] }
```

Comment:
Why `TVar`? Do you mind adding haddocks for this structure?

Reply:
`TVar` because it is shared between many server threads calling `newWork`.